### DISCUSSION TITLE
How can I use the native mmagic implementation for stable-diffusion instead of  'from_pretrained=stable_diffusion_v15_url' so that I can debug line by line through VScode?

### DISCUSSION THREAD

**Discussion by AyanKumarBhunia** [open-mmlab/mmediting#1844] [Models]
Can someone help me with the config description with the pre-trained weights link?

**Comment by Z-Fran** [comment]
`UNet2DConditionModel` and `AutoencoderKL` are from diffusers. So you need to debug in diffusers. I think you can add breakpoints in magic https://github.com/open-mmlab/mmagic/blob/main/mmagic/models/editors/stable_diffusion/stable_diffusion.py#L119 , and execute step by step until step into diffusers.  For example, if you want to debug in `UNet2DConditionModel` , you can add a breakpoints in https://github.com/open-mmlab/mmagic/blob/main/mmagic/models/editors/stable_diffusion/stable_diffusion.py#L250
